react/src/components/ProjectResourcePolicySettingModal.tsx

```diff
@@ -134,6 +136,9 @@ const ProjectResourcePolicySettingModal: React.FC<Props> = ({
             values?.max_quota_scope_size === -1
               ? -1
               : GBToBytes(values?.max_quota_scope_size),
+          ...(baiClient.isManagerVersionCompatibleWith('24.12.0') && {
```

Comment:
Please set the support checker to BAI client and use that.

Reply:
This part is weird. A boolean cannot be spread. If the manager version is above 24.12.0, this can evaluate to false.